app/templates/application.hbs

```diff
@@ -2,13 +2,22 @@
 
 <AuApp>
   <AppHeader />
-  <NavigationBar />
-
+  {{#if this.isIndex}}
+  <AuContentHeader
+    @titlePartOne="We geven je alle beslissingen"
+    @titlePartTwo="uit JOUW gemeente"
+  >
+    <img
+      sizes="50vw"
+      src={{"/assets/loket-header-1600.jpg"}}
+      alt="Foto van een laptop met daarop het vlaanderen logo."
+    />
+  </AuContentHeader>
+  {{/if}}
   <AuMainContainer as |m|>
-
     <m.content>
       {{outlet}}
-
     </m.content>
   </AuMainContainer>
+  <Footer />
```

Comment:
The footer seems to be sticky which takes up quite some space on smaller screens. I guess it makes sense on the infinite scroll page since it otherwise would keep being pushed down until the end of the list, but on the details page it might be nice if it would just scroll with the content maybe?